ast/ddl.go

```diff
@@ -14,8 +14,10 @@
 package ast
 
 import (
+	"github.com/pingcap/errors"
 	"github.com/pingcap/parser/model"
 	"github.com/pingcap/parser/types"
+	"strings"
```

Comment:
put this at line 17
and leave a blank line between the native library and the third party library